tracer/test/test-applications/debugger/Samples.Probes/SmokeTests/OverloadAndSimpleNameTest.cs

```diff
@@ -33,13 +33,22 @@ public void Run()
             new Spicing.Things.Up.OverloadAndSimpleNameTest().Method('@'); // Should be instrumented.
         }
 
-        [MethodImpl(MethodImplOptions.NoInlining | MethodImplOptions.NoOptimization)]
+        [MethodImpl(MethodImplOptions.NoInlining)]
         [MethodProbeTestData(useFullTypeName: false, unlisted: true)]
         public void Method(string callerName)
         {
             int a = callerName.Length;
             a++;
-            a++;
+            if (a > 5)
+            {
+                a++;
+            }
+            else
+            {
+                a += 2;
+            }
+
+            Console.WriteLine(a);
             a++;
```

Comment:
Have you validated that it produces the same *problematic* sequence point?

Reply:
This is not the problematic test we had